### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by rscano** [immuni-app/immuni-documentation#8]
Missing information on accessible development for people with disabilities, ensuring "accessibility by default". Accessibility as indicated by the Minister of Innovation must be an integral part of the project which must also include tests with people with different types of disabilities.

**Comment by immuniopensource** [comment]
Dear @rscano, thanks for your issue. 
Accessibility has been one of this project’s guiding principles. Immuni is implemented in iOS and Android using native technologies. As a result, Immuni can leverage everything that the operating systems offer out of the box (e.g., high text contrast, color correction).

The iOS client supports Dynamic Type, Reduced Motion, and VoiceOver. The Android client supports Font Size, Display Size, Reduced Motion, TalkBack, and Select To Speak.

The colours were chosen to ensure a sufficient contrast ratio between the text and its background, and care has been taken to ensure that colour is not the only visual means of conveying information.

Please let us know if you have further questions. Thank you!
